miniapps/navier/navier_solver.cpp

```diff
@@ -563,6 +579,15 @@ void NavierSolver::Step(real_t &time, real_t dt, int current_step,
       un_next_gf.ProjectBdrCoefficient(*vel_dbc.coeff, vel_dbc.attr);
    }
 
+   for (auto &vel_comp_dbc : vel_comp_dbcs)
+   {
+      int dim = vfes->GetVDim();
+      Coefficient** coeff_array = new Coefficient*[dim]();
+      coeff_array[vel_comp_dbc.comp] = vel_comp_dbc.coeff;
+
+      un_next_gf.ProjectBdrCoefficient(coeff_array, vel_comp_dbc.attr);
+   }
```

Comment:
This will [leak](https://godbolt.org/z/Psr45WzWr). Also, this is probably not the function you want to use to project component wise functions. See the source. There are calls to `AccumulateAndCountBdrValues` and `ComputeMeans`.